distributor/src/distributor/messenger.bal

```diff
@@ -0,0 +1,130 @@
+import ballerina/config;
+import ballerina/log;
+import ballerina/stringutils;
+import wso2/twilio;
+
+const INVALID_NO = "Invalid no";
+
+twilio:TwilioConfiguration twilioConfig = {
+    accountSId: config:getAsString("eclk.sms.twilio.accountSid"),
+    authToken: config:getAsString("eclk.sms.twilio.authToken"),
+    xAuthyKey: config:getAsString("eclk.sms.twilio.authyApiKey")
+};
+
+twilio:Client twilioClient = new(twilioConfig);
+
+// Keeps registered sms recipients in-memory. Values are populated in every service init and recipient registration
+string[] mobileSubscribers = [];
+string sourceMobile = config:getAsString("eclk.sms.twilio.source");
+
+# Send SMS notification to all the subscribers.
+#
+# + electionCode - The respective code that represents the type of election
+# + resultCode - The predefined code for a released result
+function sendSMS(string electionCode, string resultCode) {
+    string? retrievedData = getDivision(resultCode);
```

Comment:
Instead of a separate `getDivision()` function, we can do the following right?

```suggestion
    string? retrievedData = divisionCodeMap[resultCode];
```